packages/grafana-ui/src/components/TimePicker/TimeRangePicker/TimePickerContent.tsx

```diff
@@ -134,6 +134,8 @@ interface Props {
   quickOptions?: TimeOption[];
   otherOptions?: TimeOption[];
   history?: TimeRange[];
+  hideHistory?: boolean;
```

Comment:
Think it's more relevant to have `showHistory` instead, as it's relevant in the dashboard context, and most probably not so much in other parts of the UI.